pmultiqc/modules/quantms/quantms.py

```
@@ -2174,4 +2252,58 @@ def find_modification(peptide):
+        right_on="Run"
+    )
+
+    psm["Sample"] = psm["Sample"].astype(int)
+
+    mod_plot = dict()
+    for sample, group in psm.groupby("Sample", sort=True):
+
+        mod_plot_dict, _ = summarize_modifications(
+            group[["sequence", "charge", "Modifications"]].drop_duplicates()
+        )
+        mod_plot[f"Sample {str(sample)}"] = mod_plot_dict
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)